src/neptune/integrations/utils.py

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-__all__ = ["expect_not_an_experiment", "join_paths", "verify_type", "RunType"]
+__all__ = ["expect_not_an_experiment", "join_paths", "verify_type", "RunType", "SupportsNamespaces"]
```

Comment:
Do we want it to be importable from `neptune.integration.utils`? Why not just use `neptune.metadata_containers.abstract` ? Even if we want to keep the `neptune.metadata_containers.abstract` private, maybe it'd make sense to have it in some kind of "types" or "interfaces" module, it's not something I'll look for in `integration.utils`.

Reply:
Sure. I've exposed it via `neptune.typing`. What do you think?